README.md

```diff
@@ -201,3 +201,56 @@ Current swicthes and default settings:
 
 - `search-sort-radio-buttons` off - switches on/off radio selection buttons for sort order of search results.
 - `display-result-tags` off - switches on/off the display of tags in search and results pages
+
+## Makefile
+
```

Comment:
For the readme, as a user I would just be interested in:
- `make setup`
- `make run`
- `make test`

Would `make test` replace the commands in the testing section?

Reply:
I thinkg `make unit` `make integration` are also useful in addition to `make test`. But don't think we need to document all the build steps if we move this to the quickstart.

Don't think the makefile will completely replace manage.py, it just simplifies first time setup and hides some of the custom stuff we've bolted on, e.g. running sass

Reply:
Will create a new ticket for it